utils/src/main/java/org/apache/cloudstack/utils/graphite/GraphiteClient.java

```diff
@@ -67,7 +67,7 @@ protected long getCurrentSystemTime() {
      *
      * @param metrics the metrics as key-value-pairs
      */
-    public void sendMetrics(Map<String, Integer> metrics) {
+    public void sendMetrics(Map<Object, Object> metrics) {
```

Comment:
I think I missed the functional need for this type change. Can you expand @GabrielBrascher

Reply:
@DaanHoogland this method was restricted to the Graphite workflow; I needed to send another type of Objects for InfluxDB. I could create two different metrics maps or make a generic map of metrics. You can check  lines 600 and 603 of `StatsCollector.java` to see the points where I need to use the map for both cases.

Reply:
i see

                            metrics.clear();
                        }
as lines 602 and 603 but no change before it. Are these still the lines you mean?